typescript/ws-protocol/src/FoxgloveClient.ts

```diff
@@ -173,6 +192,31 @@ export default class FoxgloveClient {
     this.ws.send(payload);
   }
 
+  sendServiceRequest(request: ServiceCallRequest): void {
```

Comment:
I think this is more user friendly as otherwise the user has to fill in the correct op

```suggestion
  sendServiceRequest(request: Omit<ServiceCallRequest, "op">): void {
```

Reply:
Can also make `ServiceCallRequest` and `ServiceCallResponse` to share the base properties then